CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+19.0 (2018-04-10)
+-----------------
+
+* Allow details and summary tags (#77)
+* Add .pytest_cache to .gitignore. (#85)
+* Warn about markdown content type when checking RST (#83)
+* Update pep8 tox env to use py3.6 (#84)
+* Add Pygments-based syntax highlighting for Markdown (#82)
+* Update docs about check to talk about Markdown (#80)
```

Comment:
I'm being nitpicky here, but watch consistency in capitalization (e.g., Markdown versus markdown) in these.

Reply:
Done.